.github/workflows/VisualPinball.Engine.yml

```diff
@@ -12,5 +12,5 @@ jobs:
              dotnet build -c Release
         - name: Test
           run: |
-             .packages/NUnit.ConsoleRunner.3.11.1/tools/nunit3-console.exe VisualPinball.Engine.Test/.build/Release/VisualPinball.Engine.Test.dll
+             .packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe VisualPinball.Engine.Test/.bin/Release/net472/VisualPinball.Engine.Test.dll
```

Comment:
That's what I'll need to update the CI with, right?

Reply:
Correct.  It is using github actions now though as well, so maybe appveyor isn't needed?